serverlessworkflow/modules/ROOT/pages/security/orchestrating-third-party-services-with-oauth2.adoc

```diff
@@ -1 +1,700 @@
+
+quarkus.oidc-client.acme_financial_oauth.discovery-enabled=false <3>
+quarkus.oidc-client.acme_financial_oauth.auth-server-url=http://localhost:8281/auth/realms/kogito/protocol/openid-connect/auth <4>
+quarkus.oidc-client.acme_financial_oauth.token-path=http://localhost:8281/auth/realms/kogito/protocol/openid-connect/token <5>
+quarkus.oidc-client.acme_financial_oauth.client-id=kogito-app <6>
+quarkus.oidc-client.acme_financial_oauth.grant.type=client
+quarkus.oidc-client.acme_financial_oauth.credentials.client-secret.method=basic <7>
+quarkus.oidc-client.acme_financial_oauth.credentials.client-secret.value=secret <8>
+----
+
+<1> Is the package name for the automatically generated classes that implements the access to the all operations defined in the file `acme-financial-service.yml`.
+<2> Is the root URL to access all the operations defined in the file `acme-financial-service.yml`. For the `exchangeRate` operation, an absolute URL such as \http://localhost:8483/financial-service/exchange-rate is automatically generated.
+
+<3> Disable the Oauth2 server endpoints discovery since the endpoints provided in the <<acme-financial-service-yml, acme-financial-service.yml>> file will be used instead.
+<4> Is the authentication URL of the Oauth2 server.
+<5> Is the relative path or absolute URL of the Oauth2 token endpoint which issues access and refresh tokens.
+<6> Is the client-id to identify the Serverless Workflow against the authorization service, such as `kogito-app`. This identifier must be provided by Acme.
+<7> Is the method to use at the time to send the client-secret for the authentications when the client grant type is used.
+<8> Is the client secret to authenticate the Serverless Workflow against the authorization service when the client grant type is used. This secret must be provided by Acme.
+
+To configure the `acme_financial_service_oauth` {quarkus_guide_security_openid_connect_client_url}[Quarkus OpenId Connect Client (OIDC)] you must follow the rules described in {authentication-support-for-openapi-services-oauth-example}[Example of OAuth2 authentication].
+And the particular attributes depends on the Oauth2 server and authorization flow to use, you can get this information from the OpenAPI specification of the target service <<acme-financial-service-yml, acme-financial-service.yml>>.
+
+[NOTE]
+====
+You can use the alternatives defined in the {quarkus_guide_config_reference_url}[Quarkus configuration reference guide] to configure these properties.
+A common usage is to define {quarkus_guide_config_reference_environment_variables_url}[environment variables] to set the authentication secrets, and you can also use the {quarkus_guide_credentials_provider_url}[Quarkus Credentials Provider] framework.
+====
+
+== Running the example
+
+To run the example you must follow these steps:
+
+=== Get the example
+
+. In a new terminal, git clone the kogito-examples repository and navigate to this directory:
```

Comment:
```suggestion
. In a new terminal, clone the kogito-examples repository and navigate to this directory:
```

Reply:
In this context, "git clone" means that you clone with git. I this is fine.

Reply:
feel free to ignore it :)